assets/stylesheets/_z-index.scss

```diff
@@ -68,9 +68,9 @@ $z-layers: (
 	// but bellow #adminmenuback { z-index: 100 }
 	".edit-post-sidebar {greater than small}": 90,
 
-	// Show notices below expanded wp-admin submenus:
-	// #adminmenuwrap { z-index: 9990 }
-	".components-notice-list": 9989,
+	// Show notices below expanded editor bar
+	// .edit-post-header { z-index: 30 }
+	".components-notice-list": 29,
```

Comment:
It existed before, but I don't think it's the `NoticeList` component itself which should need to receive a `z-index`, but rather its specific usage in the editor as appearing above other content. We could reflect this by applying (merging) a class name specific to the editor notice's list `.editor-editor-notices__notice-list`

Reply:
I tried removing the z-index because I agree. But then this happens:

<img width="1074" alt="screenshot 2018-11-27 at 10 01 51" src="https://user-images.githubusercontent.com/1204802/49070257-8a250e80-f22b-11e8-90a5-ba4465d36181.png">